### DISCUSSION TITLE
MAC OS: how to avoid applying oh-my-zsh profile to my default terminal

### DISCUSSION THREAD

**Discussion by waqas-24** [ohmyzsh/ohmyzsh#10376] [Help] [answered]
is there a way I can apply oh my zsh just for iterms terminal and not to my default mac terminal? I want to keep the default mac terminal un touched and apply Oh my Zsh only to iterm terminal. Is this possible?

**Comment by mcornella** [comment, marked as answer]
You can modify your .zshrc file so that Oh My Zsh settings are only applied when `$TERM_PROGRAM = iTerm.app`, Your .zshrc file would look like this:
```zsh
# Oh My Zsh specific settings only for iTerm
if [[ "$TERM_PROGRAM" = iTerm.app ]]; then
  export ZSH=/path/to/zsh
  ZSH_THEME=<whatever>
  # other settings you might have for Oh My Zsh
  plugins=(...)
  source $ZSH/oh-my-zsh.sh
fi

# rest of the zshrc file
```

**Comment by jaredmiller219** [comment]
THIS LITERALLY SAVED MY LIFE!!!!!
